flink-autoscaler-plugin-jdbc/src/main/java/org/apache/flink/autoscaler/jdbc/event/JdbcEventInteractor.java

```diff
@@ -152,4 +153,29 @@ protected List<AutoScalerEvent> queryEvents(String jobKey, String reason) throws
     void setClock(@Nonnull Clock clock) {
         this.clock = Preconditions.checkNotNull(clock);
     }
+
+    @Nullable
+    Long queryMinEventIdByCreateTime(Timestamp timestamp) throws Exception {
+        var sql =
+                "SELECT id from t_flink_autoscaler_event_handler "
+                        + "           where id = (SELECT id FROM t_flink_autoscaler_event_handler order by id asc limit 1) "
+                        + "           and create_time < ?";
```

Comment:
`SELECT id, create_time FROM t_flink_autoscaler_event_handler order by id asc limit 1` is enough, we can check create_time in memory.

Reply:
thanks for the comment.

I conducted some tests based on the above logic, and this SQL statement has these two characteristics:
- It would run a few times  per triggering..
- Running on a large amount of data takes very little time. Even equivalent to this sentence `SELECT id FROM t_flink_autoscaler_event_handler order by id asc limit 1`

How about keeping it as now? of course, I'd like to updated it if needed.

Reply:
> How about keeping it as now? of course, I'd like to updated it if needed.

It's up to you, both of them are fine for me. 

This sql only query the first row, so the performance isn't my concern.

I leave this comment because sql looks weird at first glance.